TLM/TLM/UI/SubTools/LaneConnectorTool.cs

```diff
@@ -805,15 +805,20 @@ private class LaneEnd {
         }
 
         private static int CountLanes(ushort segmentId, ushort nodeId, bool toward) {
-            ExtSegmentManager extSegmentManager = ExtSegmentManager.Instance;
-            return extSegmentManager.GetSortedLanes(
-                                segmentId,
-                                ref segmentId.ToSegment(),
-                                ExtSegmentManager.Instance.IsStartNode(segmentId, nodeId) ^ (!toward),
-                                LaneConnectionManager.LANE_TYPES,
-                                LaneConnectionManager.VEHICLE_TYPES,
-                                true).Count;
```

Comment:
There's no point sorting lanes here, and thus no point reversing the sort order.